src/realm/util/file_mapper.cpp

```diff
@@ -111,6 +111,8 @@ std::vector<mapping_and_addr>& mappings_by_addr = *new std::vector<mapping_and_a
 std::vector<mappings_for_file>& mappings_by_file = *new std::vector<mappings_for_file>;
```

Comment:
How many of these variables should be static? file scope variables could be prefixed with "f_" (do we have a standard for that?). Then you can avoid the ugly '_' prefixes in `get_descrypted_memory_stats`.

Reply:
we don't have a standard for that, no. What's ugly with the '_' prefixes?

Reply:
but yeah, I guess they should be static.